src/components/Layout/Dashboard.tsx

```diff
@@ -12,5 +12,12 @@ export const DashboardBox = (props: BoxProps) => (
 )
 
 export const DashboardContents = (props: BoxProps) => (
-  <DashboardBox borderRadius='lg' height='100%' py={{ base: 4, lg: 10 }} px={{ base: 4, lg: 10 }} {...props} />
+  <DashboardBox
+    boxShadow='0 0 0 1px rgba(56,60,67,.05),0 1px 3px 0 rgba(56,60,67,.15)'
+    borderRadius='20px'
```

Comment:
Since we're using 20px as borderRadius in many places now, please define a proper border radius size for it.

It may be a good idea to do something similar with box shadows, since the same (complex) box shadow is being applied to multiple places (not sure about this one though).